lms/assets.py

```diff
@@ -133,20 +133,20 @@ def wrapper(context, request):
     return wrapper
 
 
-def _load_bundles(fp):
+def _load_bundles(file_):
     """Read an asset bundle config from a file object."""
     parser = configparser.ConfigParser()
-    parser.read_file(fp)
+    parser.read_file(file_)
     return {k: aslist(v) for k, v in parser.items("bundles")}
 
 
 # Site assets
-ASSETS_VIEW = static_view("lms:../build", cache_max_age=None, use_subpath=True)
-ASSETS_VIEW = _add_cors_header(ASSETS_VIEW)
+ASSETS_VIEW = _add_cors_header(
+    static_view("lms:../build", cache_max_age=None, use_subpath=True)
+)
 
 
 def includeme(config):
-    # TODO: figure out auto reload if we want it
```

Comment:
TODO and FIXME comments trigger PyLint warnings. And I actually don't think it adds much value, to have these kinds of comments littering the code, so I just deleted them.

This means you can put TODOs and FIXMEs in your WIP code for things you intend to fix before merging it, and `make lint` will be failing until you remove them